lua/lvim/core/nvimtree.lua

```diff
@@ -118,16 +129,6 @@ function M.setup()
     vim.g.netrw_banner = false
   end
 
-  local tree_cb = nvim_tree_config.nvim_tree_callback
-
-  if not lvim.builtin.nvimtree.setup.view.mappings.list then
```

Comment:
My intent was to make these 3 keybindings fully customizable, so they can also be removed by a user if he doesn't like them.
Maybe we could require nvim-tree outside the two functions to avoid requiring it two times

Reply:
> Or maybe you can use `vim.tbl_deep_extend` instead.

Fixed with `vim.list_extend()`